### ISSUE TITLE
 no such file or directory, open '/home/user/.config/Twine/story-formats.json' 

### ISSUE THREAD

**Issue by Nachosaenz** [klembot/twinejs#557]
I just downloaded the 2.3.1 version and I can't make it run.

I get this message:

./twine: /lib/x86_64-linux-gnu/libdbus-1.so.3: no version information available (required by ./twine)
/home/user/Desktop/Twine 2.3.1/twine: /lib/x86_64-linux-gnu/libdbus-1.so.3: no version information available (required by /home/user/Desktop/Twine 2.3.1/twine)
Backing up story library
ENOENT: no such file or directory, open '/home/user/.config/Twine/prefs.json'
Cannot read property 'locale' of undefined
ENOENT: no such file or directory, open '/home/user/.config/Twine/story-formats.json'

 
I'm using a Ubuntu and I was able to run 2.2.1 without problems.

**Comment by klembot** [comment]
What version of Ubuntu are you using? https://github.com/electron/electron/issues/16364 seems related, but in this bug it didn't seem to prevent startup.

Do you have a .config directory in your home directory? If not, can you try creating one (or ~/.config/Twine) and see if that resolves it?

**Comment by Nachosaenz** [comment]
Ubuntu 14.04 LTS
I have a .config file and even a .config/Twine, but the files 2.3.1 is asking for don't appear to be there.

**Comment by Nachosaenz** [comment]
Build 2.3.2, problem persists. I've tried deleting .config/Twine and letting Twine rebuild it but even then it doesn't find the files...

**Comment by kayemstar** [comment]
Let me tell you, this gave me a LOT of trouble - but I actually found a fix, elsewhere on grithub.

I'm on debian, so keep in mind I don't know how well this translates - but I created a desktop shortcut, and using that seems to avoid the issue entirely (somehow.)

Just copy pasting the instructions I myself followed. 
`#!/usr/bin/env xdg-open

[Desktop Entry]
Version=1.0
Type=Application
Terminal=false
Exec=command to run here
Name=visible name here
Comment=comment here
Icon=icon path here
`
Remember to put in the command! And if you want a working icon, make sure to install one and put in the path.

I'm still fixing OTHER problems I'm having, but it DOES load and seem functional to me. Hope this helps!
